plugins/modules/ipadnsforwardzone.py

```diff
@@ -32,53 +33,68 @@
 
 DOCUMENTATION = '''
 ---
-module: ipa_dnsforwardzone
-author: chris procter
+module: ipadnsforwardzone
+author:
+  - Chris Procter (@chr15p)
+  - Thomas Woerner (@t-woerner)
 short_description: Manage FreeIPA DNS Forwarder Zones
 description:
-- Add and delete an IPA DNS Forwarder Zones using IPA API
+  - Add and delete an IPA DNS Forwarder Zones using IPA API
 extends_documentation_fragment:
   - ipamodule_base_docs
 options:
   name:
     description:
     - The DNS zone name which needs to be managed.
+    type: list
+    elements: str
     required: true
     aliases: ["cn"]
+  action:
+    description: |
+      Work on dnsforwardzone or member level. It can be one of `member` or
+      `dnsforwardzone`.
+    type: str
+    default: "dnsforwardzone"
+    choices: ["member", "dnsforwardzone"]
   state:
     description: State to ensure
+    type: str
     required: false
```

Comment:
Should `state` don't have explicit `required: False'?

I don't see as a problem to have or not an explicit `required` field for `action` and `state`, as they always have a default, and are not required, but the documentation is not consistent among modules. My preference is to NOT have it in these two parameters.

Reply:
After offline discussion with @t-woerner (IIUC, :-) ) we should keep "required" documentation in all members.